modules/client-configuration/nav-client-configuration-guide.adoc

```diff
@@ -49,6 +49,11 @@ endif::[]
 *** xref:clients-sle.adoc[SUSE Linux Enterprise Clients]
 *** xref:clients-opensuse.adoc[openSUSE Clients]
 *** xref:clients-sleses.adoc[Expanded Support Clients]
+// Micro Clients
+*** xref:clients-slemicro.adoc[SLE Micro Clients]
+ifeval::[{uyuni-content} == true]
```

Comment:
I am not sure about this diff. If we offer SLE Micro for both Uyuni and SUSE Manager, maybe we should do the same for `openSUSE MicroOS`? Please check with @paususe

Reply:
Added a SUMA ifeval to SLE Micro. @paususe can you please weigh in here?

Reply:
@paususe ?

Reply:
I don't think we are going to support openSUSE MicroOS with SUSE Manager, at leat initially. If we are going to add the openSUSE MicroOS section, it must have a disclaimer saying it is only community-supported.

Reply:
@paususe yeah, OK, but wha about SLE Micro on Uyuni?

Reply:
SLE Micro on Uyuni: same treatment as SLES

openSUSE MicroOS: @jsrain just told me it does not actually exist as a released project/product yet, so definitely not for SUSE Manager and either nothing in Uyuni, or a disclaimer saying it's a technology preview (or something like that). The actual project/product at the moment is openSUSE Kubic.

Reply:
> openSUSE MicroOS: @jsrain just told me it does not actually exist as a released project/product yet, so definitely not for SUSE Manager and either nothing in Uyuni, or a disclaimer saying it's a technology preview (or something like that). The actual project/product at the moment is openSUSE Kubic.

Huh!? @jsrain how is that!?

https://microos.opensuse.org/ and I don't see any that shows that this is a tech preview, beta or similar. And I am using openSUSE MicroOS for almost two years and never heard about that before.

What am I missing?

Reply:
Revoking approval. This still needs to be fixed.

@Loquacity :
- SLE Micro for both SUSE Manager and Uyuni (no if)
- openSUSE MicroOS only for Uyuni (if for uyuni)

Reply:
Left the commit suggestions, I hope they are OK :-)

Reply:
fixed.